packages/widget/src/context/wallet.ts

```diff
@@ -31,37 +33,70 @@ export enum WalletContextKeys {
   SUBSTRATE_WALLET = 'substrateWallet'
 }
 
+export interface SubstrateProviderContext {
+  substrateProviders?: Array<{ domainId: number; api: ApiPromise }>;
+  selectedProvider?: ApiPromise;
```

Comment:
Why do we need this? Context consumer can obtain provider corresponding to chainId needed in it's business logic. This context shouldn't care which network is selected